### PR TITLE
fix bugs in dev (important)

### PR DESCRIPTION
**Pull request by KeweiQ**

### CODE REVIEW

File: Assets/Scripts/MoveDragon.cs [hunk 1]
```diff
@@ -175,19 +174,7 @@ public class MoveDragon : MonoBehaviour
...
      the player is on some sort of ground */
     private bool IsGrounded() {
-        // boxcast not working now
-        // float extraHeight = 0.05f;
-        // bool hitGround = Physics.BoxCast(DragonCollider.bounds.center, DragonTransform.lossyScale, DragonTransform.up * -1, Quaternion.Euler(Vector3.zero), DragonTransform.lossyScale.y + extraHeight);
         bool hitGround = Physics.Raycast(DragonTransform.position, Vector3.down, 0.5f);
-
-        Color rayColor;
-        if (hitGround) {
-            rayColor = Color.green;
```
Comment (evank28) [inline Assets/Scripts/MoveDragon.cs:185]: What is the reason for this change? Is this not needed?

**Review by evank28 — APPROVED** [review]
Playing the game, it seems to have resolved the main issue.

File: Assets/Scripts/MoveDragon.cs (hunk 1 above)
Comment (KeweiQ) [reply Assets/Scripts/MoveDragon.cs:185]: yes, this is just a test block copied from movePB.cs
